shell/platform/windows/angle_surface_manager.cc

```diff
@@ -169,18 +169,20 @@ void AngleSurfaceManager::CleanUp() {
   }
 }
 
-EGLSurface AngleSurfaceManager::CreateSurface(HWND window) {
-  if (!window || !initialize_succeeded_) {
+EGLSurface AngleSurfaceManager::CreateSurface(
+    WindowsRenderTarget* render_target) {
+  if (!render_target || !initialize_succeeded_) {
     return EGL_NO_SURFACE;
   }
 
   EGLSurface surface = EGL_NO_SURFACE;
 
   const EGLint surfaceAttributes[] = {EGL_NONE};
 
-  surface = eglCreateWindowSurface(egl_display_, egl_config_,
-                                   static_cast<EGLNativeWindowType>(window),
-                                   surfaceAttributes);
+  surface = eglCreateWindowSurface(
+      egl_display_, egl_config_,
+      static_cast<EGLNativeWindowType>(render_target->GetWindowHandle()),
```

Comment:
Since the variant only includes HWND in this change I wasn't planning to put a check in (yet).. but rather introduce it when SpriteVisual is actually added to the variant.  Resolving, please reopen if you disagree.